coding_style/README.adoc

```diff
@@ -164,6 +164,7 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   your task will likely need options such as `changed_when:` and maybe `check_mode:`).
 * Anytime `command` or `shell` modules are used, add a comment in the code with justification to help with future maintenance.
 * Use the `| bool` filter when using bare variables (expressions consisting of just one variable reference without any operator) in `when`.
+* Ensure type safety by casting variables using respective filters, such as `my_var | bool` or `my_var | int`.
```

Comment:
There is already a recommendation which is very similar to this:
```
Use float, int, and bool filters to "cast" public API variables used in numeric operations in Jinja templates
```
However, that one is more specific than yours.

Reply:
and there I was, not reading the last entry... sorry
either I can do a small change sth along the lines of:
`Use float, int, and bool filters to "cast" public API variables to ensure type safety, especially for numeric operations in Jinja` or close the PR entirely and just update the issue that it actually is obsolete.

whatever the cop prefers, it's tiny anyway.